bci_tester/data.py

```diff
@@ -319,6 +324,23 @@ def create_BCI(
         image_type="kiwi",
         bci_type=ImageType.OS,
     )
+    if TARGET == "ibs-released":
+        if False:  # not yet released
+            LTSS_CONTAINERS.append(
+                create_BCI(
+                    build_tag=f"{APP_CONTAINER_PREFIX}/ltss/sle15.3/bci-base:{OS_CONTAINER_TAG}",
+                    available_versions=["15.3"],
+                    bci_type=ImageType.OS_LTSS,
+                )
+            )
```

Comment:
Do you have an ETA for the release?

Reply:
It is waiting for the BCI team according to maintenance. Release is ready. :shrug: